ui/admin/app/templates/scopes/scope/authenticate.hbs

```diff
@@ -75,19 +78,6 @@
           {{#if @model.authMethods}}
             {{outlet}}
           {{/if}}
-          {{#unless @model.authMethods}}
```

Comment:
I removed the message which is displayed when there are no auth methods as it is not needed anymore.